src/main/java/net/imglib2/algorithm/hough/HoughTransforms.java

```diff
@@ -150,6 +153,7 @@ public static long[] getVotespaceSize( final int nRho, final int nTheta )
 	/**
 	 * Pick vote space peaks with a {@link LocalExtrema}.
 	 *
+	 * @implNote op names='features.hough.pickPeaks', type=Function
```

Comment:
Conversely from the `votespaceSize` naming question: the method here is named `pickLinePeaks`. Is it possible for there to be *non-line* peaks in a different context of Hough? If so, we should leave the Op named `features.hough.pickLinePeaks` for consistency. Otherwise, we could consider renaming the Java method.